packages/aws-cdk/test/cli/telemetry/collect-telemetry.test.ts

```diff
@@ -8,27 +8,31 @@ describe(canCollectTelemetry, () => {
   });
 
   test('returns true by default', async () => {
-    expect(canCollectTelemetry(context)).toBeTruthy();
+    expect(canCollectTelemetry({}, context)).toBeTruthy();
   });
 
   test('returns false if env variable is set', async () => {
     process.env.DISABLE_CLI_TELEMETRY = 'true';
-    expect(canCollectTelemetry(context)).toBeTruthy();
+    expect(canCollectTelemetry({}, context)).toBeTruthy();
   });
 
   test('returns false if context is set to false', async () => {
     context.set('cli-telemetry', false);
-    expect(canCollectTelemetry(context)).toBeFalsy();
+    expect(canCollectTelemetry({}, context)).toBeFalsy();
 
     context.set('cli-telemetry', 'false');
-    expect(canCollectTelemetry(context)).toBeFalsy();
+    expect(canCollectTelemetry({}, context)).toBeFalsy();
   });
 
   test('returns true if context is set to true', async () => {
     context.set('cli-telemetry', true);
-    expect(canCollectTelemetry(context)).toBeTruthy();
+    expect(canCollectTelemetry({}, context)).toBeTruthy();
 
     context.set('cli-telemetry', 'true');
-    expect(canCollectTelemetry(context)).toBeTruthy();
+    expect(canCollectTelemetry({}, context)).toBeTruthy();
+  });
+
+  test('returns false if no-version-reporting is set', async () => {
```

Comment:
`--no-version-reporting` and `--no-telemetry` are aliased -- a yargs feature. yargs will turn `--no-telemetry` into `{ "version-reporting": false }` (i.e. it won't be its own option in the yargs object) which is why there isn't another test for `returns false if no-telemetry is set.